QUICKSTART.md

```diff
@@ -381,7 +381,6 @@ ENV \
     JOB_ID=null \
     API_TOKEN=null \
     API_BASE_URL=null \
-    OS=null \
```

Comment:
these environment variable definitions should now be unnecessary. I removed them everywhere